### PR TITLE
Track domains in the title

### PR DESCRIPTION
**Pull request by NobodyNada**
[As reported by Mithrandir](https://chat.stackexchange.com/transcript/message/40861073#40861073), Metasmoke previously would not track domains that only appeared in the title of a post, since it only extracted URIs from the body of the post.  This PR does not apply the change retroactively.

### CODE REVIEW

**Comment by ArtOfCode-** [comment]
To apply retroactively: make a copy of [this migration](https://github.com/Charcoal-SE/metasmoke/blob/master/db/migrate/20170829211320_extract_domain_names_from_posts.rb) with a later timestamp.

**Comment by NobodyNada** [comment]
@ArtOfCode- Thanks; I've done that.  Seems like my commits changed `latin1` to `utf8` in schema.rb...is that bad?  Should I revert that?

File: db/schema.rb [hunk 1]
```diff
@@ -42,7 +42,7 @@
     t.index ["user_id"], name: "index_api_tokens_on_user_id"
   end
 
-  create_table "audits", id: :integer, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=latin1" do |t|
+  create_table "audits", id: :integer, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
```
Comment (angussidney) [inline db/schema.rb]: Were all of these changes intentional? IIRC this has happened in the past, but I'm not sure whether it was a good or bad thing.

**Comment by tripleee** [comment]
Linking this to https://github.com/Charcoal-SE/metasmoke/issues/261

**Comment by ArtOfCode-** [comment]
latin1 &rarr; utf8 is good, we like UTF8. utf8 &rarr; latin1 is not, which is the one that's happened before.

**Pull request merged**